### ISSUE TITLE
programming-language agnostic, not programming language-agnostic

### ISSUE THREAD

**Issue by githubshrek** [OAI/OpenAPI-Specification#2986]
The introductory paragraph currently reads:

"The OpenAPI Specification (OAS) defines a standard, programming language-agnostic interface description for HTTP APIs..."

I think it should rather read:

"The OpenAPI Specification (OAS) defines a standard, programming-language agnostic interface description for HTTP APIs..."

It currently reads as if "programming" were a separate item, as though the punctuation were "a standard, programming, language-agnostic interface description".

**Comment by karenetheridge** [comment]
no, you should read it chunked like this: "... (programming language)-agnostic ..."
